tests/L0/run_instance_norm_nvfuser/test_instance_norm_nvfuser.py

```diff
@@ -76,3 +79,24 @@ def test_sweep(self):
                 self.affine = affine
                 self.init_modules()
                 self.check_same_output() 
+
+    @unittest.skipIf(torch.cuda.device_count() < 2, "more than 1 GPU required")
+    def test_multigpu(self):
+        class Model(nn.Module):
+            def __init__(self):
+                super(Model, self).__init__()
+                self.norm = InstanceNorm3dNVFuser(4)
+
+            def forward(self, x):
+                x = self.norm(x)
+                x = torch.sum(x, dim=(1, 2, 3, 4))
+                return x
+
+        device = torch.device(f"cuda:1")
+        model = Model().to(device)
+
+        x = torch.randn(2, 4, 128, 128, 128, device=device, requires_grad=True)
+        y = torch.randn(2, device=device)
+        pred = model(x)
+        loss = nn.functional.mse_loss(pred, y.float())
+        loss.backward()
```

Comment:
could you add 

```python
if __name__ == '__main__':
    unittest.main()
```

here so that we can run this file via command line using unittest? (as is, pytest can run this test though)